templates/base.html

```diff
@@ -29,6 +29,11 @@
         {% endblock %}
     {% endblock %}
     {% block head-scripts %}{% endblock head-scripts %}
+
+    {% if environment == 'STAGING' %}
```

Comment:
We need to change this to only load it in production right? I don't think we need it for staging

Reply:
Yes, I forgot. Fixed [here](https://github.com/okfn/website/pull/656/commits/f6ba4330c1196501ddf1093fcdeb07b0378845ca)